dask/dataframe/groupby.py

```diff
@@ -1433,6 +1436,7 @@ def var(self, ddof=1, split_every=None, split_out=1):
             if not isinstance(self.index, list)
             else [self.obj] + self.index,
             chunk=_var_chunk,
+            chunk_kwargs={**self.sort},
```

Comment:
```suggestion
            chunk_kwargs=self.sort,
```
Why this syntax? To make a copy?

Reply:
I think I was probably just replacing without a ton of finesse. No need for a copy.